gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,6 @@
 #Mon Sep 14 16:21:26 CEST 2020
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https://services.gradle.org/distributions/gradle-6.6.1-bin.zip
+distributionUrl=https://services.gradle.org/distributions/gradle-6.7-rc-4-bin.zip
```

Avec alpha15 on a besoin gradle 6.7 aussi 
```suggestion
distributionUrl=https://services.gradle.org/distributions/gradle-6.7-bin.zip
```